arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -124,7 +124,9 @@ DECLARE_GAUGE(rocksdb_wal_sequence_lower_bound, uint64_t, "RocksDB WAL sequence
 DECLARE_GAUGE(rocksdb_archived_wal_files, uint64_t, "Number of archived RocksDB WAL files");
 DECLARE_GAUGE(rocksdb_prunable_wal_files, uint64_t, "Number of prunable RocksDB WAL files");
 DECLARE_GAUGE(rocksdb_wal_pruning_active, uint64_t, "Whether or not RocksDB WAL file pruning is active");
-
+DECLARE_COUNTER(arangodb_revision_tree_rebuilds_success_total, "Number of successful revision tree rebuilds");
+DECLARE_COUNTER(arangodb_revision_tree_rebuilds_failure_total, "Number of successful revision tree failures");
```

Comment:
Successful failures?
That sounds weird.

Reply:
Indeed! Thanks for spotting!